docs/tutorials/explore_view2d.ipynb

```diff
@@ -20,7 +20,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "structure_path = Path(f"{../../dynophores/tests/data/in/startframe.pdb\}")"
+    "structure_path = Path(f'{\"../../dynophores/tests/data/in/startframe.pdb\"}')"
```

Comment:
Interesting - did `flake8` do this?

Looks like an f-string but within {} we do not have as expected a variable but a string (we do not need {} in this context, since no variable).

Does this work?
```
    "structure_path = Path(\"../../dynophores/tests/data/in/startframe.pdb\")"
```

Reply:
Or in the notebook directly
```python
structure_path = Path("../../dynophores/tests/data/in/startframe.pdb")
```
(the code before as for JSON directly)

Reply:
no, it was me trying to fix flake8-nb fail "F541 f-string is missing placeholders"

Reply:
[https://github.com/dominiquesydow/dynophores/runs/2088459321](https://github.com/dominiquesydow/dynophores/runs/2088459321)
```
docs/tutorials/explore_view2d.ipynb#In[7]:5:22: F541 f-string is missing placeholders
Error: Process completed with exit code 1.
```

Reply:
Ok, I am trying to figure out why have f-strings here at all. We do not need them.

Reply:
Ah you did just that right now :)